.ci/travis/travis.sh

```diff
@@ -461,6 +461,36 @@ no-error-pmd)
   removeFolderWithProtectedFiles pmd
   ;;
 
+no-violation-test-josm)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  mkdir -p .ci-temp
+  cd .ci-temp
+  ANT_VERSION=1.10.7
+  wget -q http://www-us.apache.org/dist//ant/binaries/apache-ant-${ANT_VERSION}-bin.zip -O tmp.zip
+  unzip -o -q tmp.zip
+  rm tmp.zip
+  export ANT_HOME=$(pwd)/apache-ant-${ANT_VERSION}
+  export PATH=$ANT_HOME/bin/:$PATH
+  # Uncomment to test current tested version instead of hardcoded version
+  #TESTED=`wget -q -O - https://josm.openstreetmap.de/wiki/TestedVersion?format=txt`
+  TESTED=16391
```

Comment:
Don't you need to make sure commented TESTED is after currently uncommented TESTED. If you uncomment the top one, the bottom one will just overwrite it again.